conanfile.py

```diff
@@ -15,7 +15,8 @@ class MTConnectAgentConan(ConanFile):
     options = { "without_ipv6": [True, False], "with_ruby": [True, False], 
                  "development" : [True, False], "shared": [True, False], "winver": [None, "ANY"],
                  "with_docs" : [True, False], "cpack": [True, False], "agent_prefix": [None, "ANY"],
-                 "fPIC": [True, False], "zip_destination": [None, "ANY"] }
+                 "fPIC": [True, False], "cpack_destination": [None, "ANY"], "cpack_name": [None, "ANY"],
+                 "cpack_generator": [None, "ANY"] }
```

Comment:
Wouldn’t it be more readable if we put one property per line?

```suggestion
    options = {
        "without_ipv6": [True, False],
        "with_ruby": [True, False], 
        "development" : [True, False],
        "shared": [True, False],
        "winver": [None, "ANY"],
        "with_docs" : [True, False],
        "cpack": [True, False],
        "agent_prefix": [None, "ANY"],
        "fPIC": [True, False],
        "cpack_destination": [None, "ANY"],
        "cpack_name": [None, "ANY"],
        "cpack_generator": [None, "ANY"]
    }
```

Reply:
Agree.